stdlib/_interpchannels.pyi

```diff
@@ -0,0 +1,86 @@
+from _typeshed import structseq
+from typing import Any, Final, Literal, TypeVar, final
+from typing_extensions import Buffer, Self, TypeAlias
+
+_T = TypeVar("_T")
+_SendType: TypeAlias = Literal["send", "recv", "both"]
+
+class ChannelError(RuntimeError): ...
+class ChannelClosedError(ChannelError): ...
+class ChannelEmptyError(ChannelError): ...
+class ChannelNotEmptyError(ChannelError): ...
+class ChannelNotFoundError(ChannelError): ...
+
+@final
+class ChannelID:
+    @property
+    def end(self) -> _SendType: ...
```

Comment:
I would just use `Literal` inline here instead of defining the alias, if this is the only place where the alias is used. It's more readable for people looking through the stubs, and it'll lead to a better experience for IDE users:

```suggestion
    def end(self) -> Literal["send", "recv", "both"]: ...
```

Reply:
Will do.

Reply:
https://github.com/python/typeshed/pull/12199/commits/cf8580e6c61a1f5877605cb6bf1a685ef72bc9e2